test/unit/helpers/rubygems_helper_test.rb

```diff
@@ -52,7 +52,7 @@ class RubygemsHelperTest < ActionView::TestCase
   end
 
   should "show a nice formatted date" do
-    Timecop.travel(Time.zone.parse("2011-03-18T00:00:00-00:00")) do
+    travel_to Time.zone.parse("2011-03-18T00:00:00-00:00") do
```

Comment:
What about to test parsed time instead of `Time.zone.now` and traveling?

```ruby
time = Time.zone.parse("2011-03-18T00:00:00-00:00")
assert_equal "March 18, 2011", nice_date_for(time)
```